oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/FullGCStatsCollectorImpl.java

```diff
@@ -231,12 +238,11 @@ private static CounterStats counter(StatisticsProvider provider, String name) {
         return provider.getCounterStats(qualifiedName(name), METRICS_ONLY);
     }
 
-    private static String qualifiedName(String metricName) {
-        return FULL_GC + "." + metricName;
+    public static String qualifiedName(String metricName) {
```

Comment:
```suggestion
    private static String qualifiedName(String metricName) {
```

I don't see new usages of this method that would require to make it public.

Reply:
Fixed, thank you.